XamlControlsGallery/ControlPages/ItemsRepeaterPage.xaml

```diff
@@ -42,6 +42,49 @@
             </Button>
         </DataTemplate>
 
+        <DataTemplate x:Key="StringDataTemplate" x:DataType="x:String">
+            <Grid Background="{StaticResource SystemControlBackgroundAccentBrush}" Margin="10">
+                <TextBlock Padding="10" Text="{x:Bind}" Foreground="{StaticResource SystemControlForegroundAltHighBrush}" HorizontalAlignment="Center" TextWrapping="Wrap" VerticalAlignment="Center"></TextBlock>
```

Comment:
On an accent-colored background, we typically recommend showing white text regardless of light/dark theme. So SystemControlForegroundChromeWhiteBrush seems like a better choice.